src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Header/Content/default.html.twig

```diff
@@ -27,8 +27,10 @@
 
 {% set orderBy, orderWay = grid.sorting.order_by, grid.sorting.order_way %}
 
-{% if column.options.sortable is defined and column.options.sortable and grid.data.records_total > 0 %}
-  {{ ps.sortable_column_header(column.name, column.id, orderBy, orderWay, grid.form_prefix) }}
-{% else %}
-  {{ column.name }}
-{% endif %}
+<div class="text-{{ column.options.text_align|default('left') }}">
```

Comment:
Same here: this will modify all markup on all grids 🤔 breaking, for example, all CSS written with a cascading structure

Reply:
@jolelievre on 2019-12-12? Maybe there was a reason why he changed this? :thought_balloon:

Reply:
reverted these - most probably came from the rebase because their date is newer than the changes left on the develop branch.

Reply:
Indeed the nested ifs look bad, but right now it's fine as the template file is not too big. If it gets bigger we'll refactor it to make it look prettier